editor/editor_help.cpp

```diff
@@ -2599,25 +2846,32 @@ void EditorHelpBit::_bind_methods() {
 void EditorHelpBit::_notification(int p_what) {
 	switch (p_what) {
 		case NOTIFICATION_THEME_CHANGED: {
+			img_downloader->cancel();
 			rich_text->add_theme_color_override("selection_color", get_theme_color(SNAME("selection_color"), SNAME("EditorHelp")));
 			rich_text->clear();
-			_add_text_to_rt(text, rich_text, this);
+			_add_text_to_rt(text, rich_text, this, "", img_downloader);
 			rich_text->reset_size(); // Force recalculating size after parsing bbcode.
 		} break;
 	}
 }
 
 void EditorHelpBit::set_text(const String &p_text) {
 	text = p_text;
+	img_downloader->cancel();
 	rich_text->clear();
-	_add_text_to_rt(text, rich_text, this);
+	_add_text_to_rt(text, rich_text, this, "", img_downloader);
 }
 
 EditorHelpBit::EditorHelpBit() {
 	rich_text = memnew(RichTextLabel);
 	add_child(rich_text);
 	rich_text->connect("meta_clicked", callable_mp(this, &EditorHelpBit::_meta_clicked));
 	rich_text->set_fit_content(true);
+
+	img_downloader = memnew(EditorHelpImageDownloader);
+	img_downloader->set_rich_text_label(rich_text);
+	add_child(img_downloader);
```

Comment:
Added an extra check, closed user docs should now update correctly.

> Isn't it security risk?

If there's no vulnerability in the image loader, it will only leak IP. And providing plugins / editor scripts can execute any code, it probably should not be a big concern.

> Other than images not displaying in EditorHelpBit

It can be added, but I'm not sure if adding images to the tooltip is useful.

Reply:
I agree it's not useful, I only meant that the error should be fixed. Now there is simply a loading icon, which I guess is acceptable?
![image](https://github.com/godotengine/godot/assets/2223172/65c6d75f-0477-43d5-b02a-7922a4639610)